client/my-sites/upgrades/domain-management/edit-contact-info/form-card.jsx

```diff
@@ -118,6 +120,26 @@ class EditContactInfoFormCard extends React.Component {
 		}
 	}
 
+	hasEmailChanged() {
+		return this.props.contactInformation.email === formState.getFieldValue( this.state.form, 'email' );
+	}
+
+	getCurrentEmails() {
+		const currentEmail = this.props.contactInformation.email,
+			wpcomEmail = this.props.currentUser.email;
+
+		if ( currentEmail === wpcomEmail ) {
+			return currentEmail;
+		}
+
```

Comment:
:information_source: This string is probably long enough that it doesn't need a **context**. Are you sure you don't want to use a translator comment instead?